package.json

```diff
@@ -59,6 +59,8 @@
   "dependencies": {
     "@types/jest": "^27.0.1",
     "@types/systemjs": "^6.1.1",
+    "react-date-picker": "^8.3.2",
```

Comment:
move these to the components you used them in

Reply:
No be used am na